webapp/src/main/resources/db/migration/V64__third_party_sync_file_checksum.sql

```diff
@@ -0,0 +1,4 @@
+create table third_party_sync_file_checksum (repository_id bigint not null, locale_id bigint not null, checksum char(32) not null, file_name varchar(255) not null);
```

Comment:
Btw, just noticed that every other md5 column definition is (incorrectly) using varchar(32) instead of char(32) - which is the recommended type. Nice that we're using the correct type here!

Reply:
I would also recommend having _created_date_ & _last_modified_date_ columns for debuggability/tracking purposes and have the entity for the table extend: SettableAuditableEntity.

Reply:
Even for mapping tables - as a best practice we should assign every row a surrogate key:
`id           BIGINT   NOT NULL AUTO_INCREMENT,`
[..]
`PRIMARY KEY (id)`